config/samples/kustomization.yaml

```
@@ -4,4 +4,5 @@ resources:
 - netbox_v1_ipaddressclaim.yaml
 - netbox_v1_prefix.yaml
 - netbox_v1_prefixclaim.yaml
+- netbox_v1_prefixclaim_customfields.yaml
```

```suggestion
- netbox_v1_prefixclaim_parentprefixselector_bool_int.yaml
- netbox_v1_prefixclaim_parentprefixselector.yaml
```
Just noticed that the filenames changed :)

---

Good catch! :) Thanks